README.md

```diff
@@ -777,15 +777,35 @@ oneview_server_profile_template 'ServerProfileTemplate1' do
   ethernet_network_connections <ethernet_network_connections_data>
   fc_network_connections <fc_network_connections_data>
   network_set_connections <network_set_connections_data>
-  server_profile_name <profile_name>  # String - Optional. In :new_profile action represents the name of the Server Profile to created. In create/create_if_missing is the name of the Server Profile be used as base for the Server Profile Template. Only available on API500 and onwards to be used as base.
+  server_profile_name <profile_name>           # String - Optional. In :new_profile action represents the name of the Server Profile to created. In create/create_if_missing is the name of the Server Profile be used as base for the Server Profile Template. Only available on API500 and onwards to be used as base.
   os_deployment_plan <os_deployment_plan_name>
+  volume_attachments <volume_attachments_data> # Array<Hash> - The volume attachments data to be created with Server Profile Template.
   action [:create, :create_if_missing, :delete, :new_profile]
 end
 ```
 
 You can specify the association of the server profile with each of the resources using the resource properties. Also it is easy to add connections using the connection properties:
 
 - **\<resource_name\>_connections** (Array/Hash) Optional - Specify connections with the desired resource type. The Hash entry should have `<network_name> => <connection_data>` associations. The Array contains these Hash entries. See the examples for more information.
+- **volume_attachments** (Array<Hash>) Optional - Specify a list of volume attachments to be created when create/update the Server Profile Template. See the [example](examples/server_profile_template.rb) for more information.
+
+  To attach a existent Volume use a data as the below to be the volume attachment data:
+  ```ruby
+  {
+    volume: 'test2', # name of existent Oneview Volume
+    attachment_data: {} # # key-pair data to be the specific attributes of Oneview Volume Attachment
+  }
+  ```
+  To create a new Volume and attach it, use a data as the below to be the volume attachment data:
+  ```ruby
+    {
+      volume_data: {}, # key-pair data to create a new Volume to the Oneview
+      storage_system: 'ThreePAR-1', # name of Storage System associated with the Volume Attachment
+      storage_pool: 'CPG-SSD', # name of Storage Pool associated with the Volume Attachment
+      host_os_type: 'Windows 2012 / WS2012 R2', # the hostOsType info of San Storage
+      attachment_data: {} # key-pair data to be the specific attributes of Oneview Volume Attachment
```

Comment:
key-pair data to be the specific attributes of **the** Oneview Volume Attachment